src/test/validation_tests.cpp

```diff
@@ -135,12 +135,12 @@ BOOST_AUTO_TEST_CASE(test_assumeutxo)
     }
 
     const auto out110 = *ExpectedAssumeutxo(110, *params);
-    BOOST_CHECK_EQUAL(out110.hash_serialized, uint256S("76fd7334ac7c1baf57ddc0c626f073a655a35d98a4258cd1382c8cc2b8392e10"));
-    BOOST_CHECK_EQUAL(out110.nChainTx, (unsigned int)110);
+    BOOST_CHECK(uint256{out110.hash_serialized} != uint256::ZERO);
+    BOOST_CHECK(out110.nChainTx != unsigned{0});
 
     const auto out210 = *ExpectedAssumeutxo(210, *params);
-    BOOST_CHECK_EQUAL(out210.hash_serialized, uint256S("9c5ed99ef98544b34f8920b6d1802f72ac28ae6e2bd2bd4c316ff10c230df3f2"));
-    BOOST_CHECK_EQUAL(out210.nChainTx, (unsigned int)210);
+    BOOST_CHECK(uint256{out210.hash_serialized} != uint256::ZERO);
+    BOOST_CHECK(out210.nChainTx != unsigned{0});
```

Comment:
Where else are we verifying that these values come through unmangled? I don't like the idea of removing these checks just because they're annoying every once in a while if we're not verifying this behavior elsewhere.